import-dashboards.py

```diff
@@ -242,6 +244,47 @@ def copy_apps():
             shutil.copytree(source_dir, dest_dir)
 
 
+def create_folders():
+    for folder in SET_OF_TAGS.keys():
+        path = PMM_PLUGIN_DIR + folder
+        try:
+            os.mkdir(path)
+        except OSError:
+            print ' * Creation of subfolder %s failed' % path
+        else:
+            print ' * Successfully created subfolder %s' % path
+
+
+def move_dashboards_into_folders():
+    for filename in os.listdir(PMM_PLUGIN_DIR):
+        if os.path.isfile(os.path.join(PMM_PLUGIN_DIR, filename)):
+            with open(PMM_PLUGIN_DIR + filename, 'r') as dashboard_file:
+                dashboard = json.loads(dashboard_file.read())
+                tag = dashboard['tags'][0]
+                if tag == 'Percona':
+                    try:
+                        tag = dashboard['tags'][1]
+                    except:
+                        continue
+                try:
+                    shutil.move(os.path.join(PMM_PLUGIN_DIR, filename), PMM_PLUGIN_DIR + tag)
+                    print ' * Moved dashboard %s into subfolder %s' % (dashboard['title'], PMM_PLUGIN_DIR + tag)
+                except:
+                    continue
+
+
+def create_provisioning_configs():
+    for folder in SET_OF_TAGS.keys():
+        try:
+            f = open(GRAFANA_PROVISION_DIR + folder + '.yaml',"w+")
+            f.write('apiVersion: 1\n\nproviders:\n - name: %s\n   orgId: 1\n   folder: %s\n   type: file\n   updateIntervalSeconds: 360\n' % (folder,folder))
+            f.write('   options:\n     path: %s\n' % ('/var/lib/grafana/plugins/pmm-app/dist/dashboards/' + folder))
+            f.close()
+            print ' * Created provisioning config file %s.yaml' % (GRAFANA_PROVISION_DIR + folder)
+        except:
+            continue
```

Comment:
`continue` looks hear rudimental. What do you think to print what error happened? Could be useful for debug.

Reply:
Added print out of an exception